### DISCUSSION TITLE
[Q] How to add dropdown filters in first row of an excel based on known value list or automatically gathered list from column values

### DISCUSSION THREAD

**Discussion by moda20** [exceljs/exceljs#2384] [Q&A]
i want to add a column header filter using a dropdown list in excel (xlsx), 
something like  this image : 
https://v1.nitrocdn.com/zciqOWZgDznkuqFpvIJsLUeyHlJtlHxe/assets/static/source/rev-029371b/wp-content/uploads/2013/07/Custom-Filter-Image.gif

**Comment by yogithesymbian** [comment]
its look like these ? https://github.com/exceljs/exceljs/discussions/2202
